docs/book/src/migration/multi-group.md

```diff
@@ -4,32 +4,35 @@
 
 <h1>Note</h1>
 
-Multi-group scaffolding support was not present in the initial version of
-the Kubebuilder v2 scaffolding (as of Kubebuilder v2.0.0).
-
-To change the layout of your project to support Multi-Group run the command
-`kubebuilder edit --multigroup=true`. Once you switch to a multi-group layout, the new Kinds
-will be generated in the new layout but additional manual work is needed 
-to move the old API groups to the new layout.
-
-</aside>
-
-While Kubebuilder v2 will not scaffold out a project structure compatible
+While Kubebuilder will not scaffold out a project structure compatible
 with multiple API groups in the same repository by default, it's possible
 to modify the default project structure to support it.
 
+</aside>
+
 Let's migrate the [CronJob example][cronjob-tutorial].
 
 <aside class="note warning">
-<h1>Using go/v4</h1>
+<h1>Instructions vary per project layout</h1>
+
+You can verify the version by looking at the PROJECT file. The currently default and
+recommended version is go/v4.
 
-If you create your project using go/v4 plugin (you can verify it by looking at the PROJECT file )
-then, all steps are the same but you need to keep in mind that the api and controllers directory
-path is now under the `pkg` directory instead. So, you need ensure that you update the
-paths accordingly.
+The layout go/v3 is **deprecated**, if you are using go/v3 it is recommended that you migrate to
+go/v4, however this documentation is still valid. [(More Info)][migration-guide].
```

Comment:
Just a nit it is missing add the alias at the bottom, see in the preview: https://deploy-preview-3403--kubebuilder.netlify.app/migration/multi-group.html
![Screenshot 2023-05-16 at 22 04 57](https://github.com/kubernetes-sigs/kubebuilder/assets/7708031/3fc1ad3e-dac3-4cdc-b735-d4679e6561f6)